demo/index.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
```

Comment:
I would love to keep the demo as simple as possible (with just the events).
Do you mind reverting these changes?

Reply:
Oh sure, removed them.

Reply:
Oh sure, reverted as original.